README.md

```diff
@@ -4,14 +4,50 @@
-diffusions. It is one of the core solver libraries included in [DifferentialEquations.jl](https://github.com/JuliaDiffEq/DifferentialEquations.jl). 
-Users interested in using this functionality should see the 
-[DifferentialEquations.jl documentation](https://diffeq.sciml.ai/latest/). The documentation includes 
-- [a tutorial and details on using DiffEqJump to simulate jump processes via SSAs (i.e. Gillespie methods)](https://diffeq.sciml.ai/latest/tutorials/discrete_stochastic_example/), 
-- [a reference on the types of jumps and available simulation methods](https://diffeq.sciml.ai/latest/types/jump_types/), 
-- [a FAQ](https://diffeq.sciml.ai/latest/tutorials/discrete_stochastic_example/#FAQ) with information on changing parameters between simulations and using callbacks.
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](http://jump.sciml.ai/stable/)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](http://jump.sciml.ai/dev/)
+
+DiffEqJump.jl provides methods for simulating jump processes, known as
+stochastic simulation algorithms (SSAs), Doob's method, Gillespie methods, or
+Kinetic Monte Carlo methods across different fields of science. It also enables the
+incorporation of jump processes into hybrid jump-ODE and jump-SDE models,
+including jump diffusions.
+
+DiffEqJump is a component package in the [SciML](https://sciml.ai/) ecosystem,
+and one of the core solver libraries included in
+[DifferentialEquations.jl](https://github.com/JuliaDiffEq/DifferentialEquations.jl).
+
+The documentation includes
+- [a tutorial and details on using DiffEqJump to simulate jump processes via SSAs (i.e. Gillespie methods)](https://jump.sciml.ai/latest/discrete_stochastic_example/),
+- [a reference on the types of jumps and available simulation methods](https://diffeq.sciml.ai/latest/jump_types/),
+- [a FAQ](https://diffeq.sciml.ai/latest/faq.md) with information on changing parameters between simulations and using callbacks.
+
+## Installation
+There are two ways to install `DiffEqJump.jl`. First, users may install the meta
+`DifferentialEquations.jl` package, which installs and wraps `OrdinaryDiffEq.jl`
+for solving ODEs, `StochasticDiffEq.jl` for solving SDEs, and `DiffEqJump.jl`,
+along with a number of other useful packages for solving models involving ODEs,
+SDEs and/or jump process. This single install will provide the user with all of
+the facilities for developing and solving Jump problems.
+
+To install the `DifferentialEquations.jl` package, refer to the following link
+for complete [installation
+details](https://docs.sciml.ai/dev/modules/DiffEqDocs/).
+
+If the user wishes to separately install the `DiffEqJump.jl` library, which is a
+lighter dependency than `DifferentialEquations.jl`, then the following code will
+install `DiffEqJump.jl` using the Julia package manager:
+```julia
+using Pkg
+Pkg.add("DiffEqJump")
+```
+
```

Comment:
we usually put one nice looking example in the README

Reply:
I plan to give it a more thorough update before I merge this.

Reply:
as in turning on docs examples with strict mode on?

Reply:
Up to you how much we want to update it / clean it up before proceeding. i.e. I can convert everything to examples, add strict mode, split the tutorials, get all the API references in and merged,  merge/reconcile the jump and solver type documentation with the doc strings, etc... Then we can merge and release, but probably this will take me till the end of the week since I have other stuff I need to work on the next few days (including the Catalyst paper draft).

Alternatively, we can go with this, which is a more straight conversion of the DifferentialEquations.jl docs, and then I can work on improving it over the next week or so.

I'm running this PR as a branch off the main repo, so we can view a preview build on the website to see how it'll look.

Reply:
Straight conversion and then continue is probably best